mne/channels/channels.py

```diff
@@ -403,6 +403,8 @@ def pick_channels(self, ch_names, copy=False):
         -----
         .. versionadded:: 0.9.0
         """
+        if not isinstance(ch_names, (tuple, list, np.ndarray)):
+            ch_names = [ch_names]
```

Comment:
I'd have raised an error if ch_names is "not isinstance(ch_names, (tuple, list, np.ndarray))". Otherwise you need to update the docstring saying you access str and list???

what do others think?